### PR TITLE
Reorganize Dashboards

### PR DESCRIPTION
**Pull request by Daltz333**
Partial Successor to #993 

Marking as ``Needs Design Decision`` to see if we want to make these changes mid-season. (Analyze impact)

Preview: https://frc-docs--1142.org.readthedocs.build/en/1142/

### CODE REVIEW

**Comment by sciencewhiz** [comment]
Can you describe the structure of the reorganization? I'm having trouble figuring out what it is among the 193 changes.

**Comment by Daltz333** [comment]
Glass, Shuffleboard, and Smartdashboard is in a root TOC "Dashboards" section. That's it.

**Comment by jasondaming** [comment]
Reorder the dashboards in the TOC shuffleboard, smartdashboard, glass, LV and I am good with this.

File: source/redirects.txt [hunk 1]
```diff
@@ -84,9 +84,9 @@
 "docs/zero-to-robot/step-3/creating-benchtop-test-program-labview.rst" "docs/zero-to-robot/step-4/creating-benchtop-test-program-labview.rst"
 "docs/zero-to-robot/step-3/running-benchtop-test.rst" "docs/zero-to-robot/step-4/running-benchtop-test.rst"
 "docs/software/old-commandbased/commands/limit-switches-control-behavior.rst" "docs/software/sensors/limit-switch.rst"
-"docs/software/driverstation/driver-station-labview-dashboard.rst" "docs/software/labview-dashboard/driver-station-labview-dashboard.rst"
-"docs/software/driverstation/troubleshooting-dashboard-connectivity.rst" "docs/software/labview-dashboard/troubleshooting-dashboard-connectivity.rst"
-"docs/software/driverstation/using-the-labview-dashboard-with-c++-java-code.rst" "docs/software/labview-dashboard/using-the-labview-dashboard-with-c++-java-code.rst"
```
Comment (TheTripleV) [inline source/redirects.txt:89]: redirects.txt shouldn't have stuff removed from it (or be modified in the middle).

**Review by jasondaming — APPROVED** [review]

**Pull request merged**